include/MidiClip.h

```diff
@@ -91,6 +91,11 @@ class LMMS_EXPORT MidiClip : public Clip
 		return m_clipType;
 	}
 
+	inline int steps() const
```

Comment:
`inline` is redundant when you are defining the member function within the `class` declaration.

```suggestion
	int steps() const
```